src/pyaerocom_preproc/s3_bucket.py

```diff
@@ -28,6 +28,17 @@ def s3_upload(path: Path, *, object_name: str | None = None):
     if object_name is None:
         object_name = path.name
     try:
-        s3_client(settings).upload_file(path, settings.s3_bucket.bucket_name, object_name)
+        s3_client(settings).upload_file(
+            path, settings.s3_bucket.bucket_name.strip("s3://"), object_name
+        )
     except ClientError as e:
         logger.error(f"{e}, skip")
+
+
+def s3_list():
+    if (settings := config()) is None:
+        raise Abort()
+    hostname = settings.s3_bucket.bucket_name.strip("s3://")
```

Comment:
I think that the leading "s3://" should be stripped from the credentials hen they are inputted, or even better they should fail the credential check stage.

Reply:
Good call. I will add a check to see if this appears in the bucket_name and inform the user to overwrite the bucket_name if it does. We also will need to remember to inform the users of how to format this field